packages/wrangler/src/deployment-bundle/entry.ts

```diff
@@ -25,6 +25,8 @@ export type Entry = {
 	file: string;
 	/** A worker's directory. Usually where the Wrangler configuration file is located */
 	projectRoot: string;
+	/** The path to the config file, if it exists. */
+	configPath: string | undefined;
```

Comment:
Does this need to be in `Entry`? Is it not available otherwise?

Reply:
It's not cleanly available to the `findAdditionalModules()` machinery, similar to to projectRoot.